src/experiments/webgpuaudio/ir-helper.js

```diff
@@ -25,6 +25,30 @@ const createTestIR = (length) => {
   return impulseResponse;
 }
 
-export default {
+/**
+ * Produces a new Float32Array object from an audio file at the given URL.
+ * Since this requires a valid AudioContext, this function can only gets called
+ * from the main thread.
+ * 
+ * @param {AudioContext} audioContext An AudioContext instance.
+ * @param {string} url A URL to the IR audio file.
+ * @returns {Float32Array}
+ * 
+ * @example
+ * const irArray = await fetchAudioFileToF32Array(
+ *     audioContext,
+ *     '../../sounds/impulse-responses/cardiod-35-10-spread.wav');
+ * console.log(array);    // You'll see a big array!!! 
+ */
+const fetchAudioFileToF32Array = async (audioContext, url) => {
```

Comment:
nit: I wonder if we should just rename this file to helper.js, or audio-helper.js, since this is also doing more than just returning the impulse response. What do you think?

Reply:
Fair point, but currently both functions are for creating or loading IR files. If the scope expands in the future, I'll definitely change the name.